features/host_can_view_navbar.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Host can view navbar and see "Become a Host" link 
+    As a Host 
+    In order to use the service
+    I would like to navigate the site
+
+  @javascript
+  Scenario: Visitor can see the landing page
+    When I visit the "landing page"
```

Comment:
Maybe we should change "landing page" to without string - i.e landing page - to be consistent with other PRs?

Reply:
Changed!